internal/server/iigointernal/utilities.go

```diff
@@ -67,6 +67,10 @@ func WithdrawFromCommonPool(value int, gameState *gamestate.GameState) error {
 	}
 }
 
+func depositIntoCommonPool(value int, state *gamestate.GameState) {
```

Comment:
This should be a server method, usually when you feel the need to pass a pointer around consistently, it's a good time to ask "should I instead make this a method of an existing struct?"

It's fine for now (if it works), just a note.

Reply:
all the ints referring to resources in this PR should also be `shared.Resources`.

Reply:
Hmmm it's a good question you're asking and one of the reasons we had this sort of separate was to organise it a little better with the number of files we had. We've got another PR open (#138 ) that addressed the shared resources issue. Therefore I will close this one.